addons/rating/views/rating_templates.xml

```diff
@@ -3,15 +3,19 @@
         <!-- External page : thanks message -->
         <template id="rating_external_page_view" name="Thanks for your Feedback">
             <t t-call="web.frontend_layout">
-                <div class="container pt-5">
-                    <div class="oe_structure" id="oe_structure_rating_header_thanks_you"/>
-                    <div class="text-center mt-5 pt-5">
-                        <h3 class="d-inline">Thank you, we appreciate your feedback!</h3>
+                <div class="o_rating_page_view d-flex">
```

Comment:
I would maybe use `d-flex flex-column justify-content-center` on the parent (`.o_rating_page_view`) instead of adding `my-auto` on the child block (`.container`). That way, people can add new elements to `o_rating_page_view` and those elements will be properly centered as well :smile:

Reply:
I took the oe_structure_rating_footer_thanks_you into account, but sure, why not being extra careful :) Change done